syntax = "proto3";

package vectordb_rpc;

service VectorDB {
  rpc Ping (PingRequest) returns (PingReply) {}
  rpc Info (InfoRequest) returns (InfoReply) {}
}

message PingRequest {
  string msg = 1;
}

message PingReply {
  string msg = 1;
}

message InfoRequest {
}

message InfoReply {
  string msg = 1;
}
